lib/src/clixon_options.c

```diff
@@ -354,6 +357,9 @@ parse_configfile(clicon_handle  h,
                 /* Append to master (removed from xe) */
                 if (xml_addsub(xt, xec) < 0)
                     goto done;
+                continue;
+ignore:
```

Comment:
Please avoid goto:s unless it is to the single return, see:
https://github.com/clicon/clixon/blob/master/CONTRIBUTING.md#return-values

Also please extend the test https://github.com/clicon/clixon/blob/master/test/test_configdir.sh if possible with the usecase that causes a loop. In that way it is added to the regression.

Reply:
Done